NftVotePlugin/NftProposalVoteState.tsx

```diff
@@ -32,9 +34,9 @@ const NftProposalVoteState = ({
 
   const showVoteRecords =
     isReady &&
-    voterWeight &&
+    calculatedMaxVoterWeight?.value &&
```

Comment:
should this be calculatedVoterWeight? (same below)

Reply:
Yes, wrong object.